Sources/BraintreeVenmo/BTVenmoDriver.m

```diff
@@ -159,6 +159,9 @@ - (void)tokenizeVenmoAccountWithVenmoRequest:(BTVenmoRequest *)venmoRequest comp
 
             [self.apiClient POST:@"" parameters:params httpType:BTAPIClientHTTPTypeGraphQLAPI completion:^(BTJSON *body, __unused NSHTTPURLResponse *response, NSError *err) {
                 if (err) {
+                    if ([err.localizedDescription isEqualToString:@"The network connection was lost."]) {
```

Comment:
Is there another check we can make to see if it's a _network connection error_ that doesn't rely on a `"network connection was lost."` string?

Reply:
Yep, looks like we can check the `error.code` instead. lldb was losing it's mind when we were trying to do that yesterday, but it looks like it's fine now and Xcode is happy letting us do that. 🤦

Reply:
Nice yeah that'd be dope in case that string changes.

Reply:
Yup! Updated! Those this error hasn't changed since I think Swift dropped so probably a lot of things would blow up if they ever changed that string (but now we won't be one of them).